lib/dest/options.js

```diff
@@ -29,6 +33,18 @@ var config = {
     type: ['string', 'boolean'],
     default: false,
   },
+  // Symlink options
+  useJunctions: {
+    type: 'boolean',
+    default: function(file) {
+      var isDirectory = file.isDirectory();
+      return (isWindows && isDirectory);
```

Comment:
Yeah, this should be a separate PR though, as it's not as simple as changing the default